frontend/src/app/main/ui/shapes/attrs.cljs

```diff
@@ -193,9 +194,23 @@
 
          styles (-> (obj/get props "style" (obj/new))
                     (obj/merge! svg-styles)
-                    (add-fill shape render-id)
                     (add-stroke shape render-id)
-                    (add-layer-props shape))]
+                    (add-layer-props shape))
+
+         styles (cond (or (some? (:fill-image shape))
+                          (= :image (:type shape))
+                          (> (count (:fills shape)) 1)
+                          (some #(some? (:fill-color-gradient %)) (:fills shape)))
```

Comment:
You can just use `(some :fill-color-gradient)`

Reply:
Do you refer just to the last condition so it looks like this?
```
(some :fill-color-gradient (:fills shape))
```

Reply:
Yes